apps/e2e/integration/note-transactions/outbound_note_transactions.spec.ts

```diff
@@ -942,8 +941,8 @@ test("should show no warehouse selected after the assigned warehouse is deleted"
 	// Action: Delete the assigned warehouse.
 	await dbHandle.evaluate(deleteWarehouse, 1);
 
-	// Assertion: Verify the transaction row now shows no warehouse selected.
-	await entries.assertRows([{ isbn, quantity: 1, warehouseName: "" }]);
+	// Assertion: Verify the transaction row now does not exist
+	await entries.assertRows([]);
```

Comment:
Do we want this behaviour? 
I think it makes sense to clear all In Note's and their transactions that are associated with the warehouse, but an Out Note transaction could still exist, and a user could just select a new option? 
I would say this calls for two updates:
- Update the warehouse delete dialog message to notify the user that all associated Purchase notes will be deleted
- Update the "deleteWarehouse" handler so that book_transaction rows are not deleted if they are for outbound notes...
- and that this behaviour still stands, with an outbound note row showing "no warehouse selected" after